### ISSUE TITLE
Add a rule for irs.gov

### ISSUE THREAD

**Issue by sanctiondt** [apple/password-manager-resources#315]
Add a rule for irs.gov

<img width="761" alt="Screen Shot 2020-08-14 at 10 21 44 AM" src="https://user-images.githubusercontent.com/61439755/90683872-18210400-e21c-11ea-82d8-f12af3126b03.png">

**Comment by mleibner** [comment]
@hisaac That's correct, the updates in pull request #278 for www.4.irs.gov should address the screenshot shared by @sanctiondt (that screenshot is either from sa.www4.irs.gov or la.www4.irs.gov).

Based on the screenshot from @sanctiondt , though, could that mean the rule from #278 might not be working properly?

**Comment by hisaac** [comment]
@sanctiondt Could you share some more specifics on the URL you're seeing this at message at?

**Comment by mleibner** [comment]
@hisaac The URL isn't accessible directly, but that page from the screenshot @sanctiondt provided would either be `https://sa.www4.irs.gov/eauth/pub/registration/profile_create0` or `https://la.www4.irs.gov/eauth/pub/registration/profile_create0` (depends on which specific IRS application @sanctiondt was accessing).

If you need to access that page for testing purposes, the easiest way is via the IRS [Form 990-N Electronic Filing System (e-Postcard)](https://sa.www4.irs.gov/epostcard/) site - click "Create Account", submit your name & email address, confirm the code sent via email, and then you'll get to that `profile_create0` page with the password creation box.

**Comment by sanctiondt** [comment]
Correct. In my case it was the sa.ww4.irs.gov URL.

**Comment by rmondello** [comment]
@sanctiondt It feels like maybe you were using a build of Safari that didn’t have the project’s rule. What version of Safari were you using?

**Comment by sanctiondt** [comment]
Standard customer build: Version 13.1.2 (15609.3.5.1.3)

**Comment by rmondello** [comment]
Then it wouldn’t have shipped yet. I think we’re good here! Thanks for the help @mlabuhn and @hisaac!
